jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -0,0 +1,654 @@
+    "                num_loops (int): number of times to poll before failing\n",
+    "    Prints the state of the order\n",
+    "    '''\n",
+    "    state = ''\n",
+    "    end_states = ['success', 'failed', 'partial']\n",
+    "    while state not in end_states:\n",
+    "        r = requests.get(order_url, auth=session.auth)\n",
+    "        response = r.json()\n",
+    "        state = response['state']\n",
+    "        print(state)\n",
+    "        if state in end_states:\n",
+    "            break\n",
+    "        time.sleep(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "queued\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "success\n"
+     ]
+    }
+   ],
+   "source": [
+    "poll_for_success(order_url)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_provenance_raster_merge_clip.tif\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_merge_clip.tif\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_provenance_vector_merge_clip.zip\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_metadata_merge_clip.json\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_ortho_udm_merge_clip.tif\n",
+      "basemaps-to-cloud/31c7dd26-c8a0-4897-a56f-83e1ef0cf40f/manifest.json\n"
+     ]
+    }
+   ],
+   "source": [
+    "r = requests.get(order_url, auth=session.auth)\n",
+    "response = r.json()\n",
+    "results = response['_links']['results']\n",
+    "\n",
+    "print(\"\\n\".join([r['name'] for r in results]))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use Basemaps API to get the Basemap and quad IDs\n",
```

Comment:
I believe all my feedback from this section on the other notebook applies in the following section as well - use `session` instead of `request`, and revise your error handling to account for 4xx error codes.